_includes/0.2/left_sidebar.html

```diff
@@ -40,10 +40,12 @@
         <a href="{% link docs/0.2/creating_schemas.md %}">Creating Schemas</a>
         <a href="{% link docs/0.2/creating_products.md %}">Creating Products</a>
         <a href="{% link docs/0.2/creating_locations.md %}">Creating Locations</a>
+        <a href="{% link docs/0.2/upgrading_smart_contracts_for_administrators.md %}">Upgrading Smart Contracts For Administrators</a>
```

Comment:
Maybe remove the _for_administrators and _for_developers from the titlees since it is already in the heading? Although it could be confusing to have 2 different docs with the same name

Reply:
I considered this but I think it's more clear with them